api/src/main/java/io/strimzi/api/kafka/model/acl/AclOperation.java

```diff
@@ -2,7 +2,7 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.api.kafka.model;
+package io.strimzi.api.kafka.model.acl;
```

Comment:
just an idea, doesn't it make more sense having under `user` package so ... `io.strimzi.api.kafka.model.user.acl`?

Reply:
Yeah that was what we thought at start. But if oyu go throiugh this sheet -> https://docs.google.com/document/d/1uZwYGZFo9FIR-95q2PV5Npl6IN1KfWSr12cRp21VMpM/edit, I made this change based on Tom's comment

Reply:
@ppatierno WDYT about this? Do you want @ShubhamRwt to change or or are you fine with it?

Reply:
Tbh I didn't get what @tombentley meant with that comment :-)

Reply:
@tombentley Hi, should we have the acl's under user package?